eZ/Publish/Core/SignalSlot/ContentService.php

```diff
@@ -323,19 +323,25 @@ public function deleteContent(ContentInfo $contentInfo)
      * @param \eZ\Publish\API\Repository\Values\Content\ContentInfo $contentInfo
      * @param \eZ\Publish\API\Repository\Values\Content\VersionInfo $versionInfo
      * @param \eZ\Publish\API\Repository\Values\User\User $user if set given user is used to create the draft - otherwise the current user is used
+     * @param \eZ\Publish\API\Repository\Values\Content\Language|null if not set the draft is created with the initialLanguage code of the source version or if not present with the main language.
      *
      * @return \eZ\Publish\API\Repository\Values\Content\Content - the newly created content draft
      */
-    public function createContentDraft(ContentInfo $contentInfo, VersionInfo $versionInfo = null, User $user = null)
-    {
-        $returnValue = $this->service->createContentDraft($contentInfo, $versionInfo, $user);
+    public function createContentDraft(
+        ContentInfo $contentInfo,
+        VersionInfo $versionInfo = null,
+        User $creator = null,
+        ?Language $language = null
+    ) {
+        $returnValue = $this->service->createContentDraft($contentInfo, $versionInfo, $creator, $language);
```

Comment:
I know that `$returnValue` is the old name but i think `$contentDraft` could be better. Than we can change `$returnValue->getVersionInfo()->initialLanguageCode` to `$contentDraft->getVersionInfo()->initialLanguageCode`